### ISSUE TITLE
Right side blue focus outline of chat is missing if we maximize the chat panel

### ISSUE THREAD

**Issue by abhijit-chikane** [microsoft/vscode#229999]
![image](https://github.com/user-attachments/assets/85a1fa69-3ebc-432e-a8ea-c44b9a67629f)

right side blue outline of the missing if we maximize the chat panel

**Comment by roblourens** [comment]
I can't repro- what do you mean by "maximize the chat panel"? And can you share the theme?

**Comment by roblourens** [comment]
We also just shipped a new release so please check if it reproes in the latest

**Comment by abhijit-chikane** [comment]
![image](https://github.com/user-attachments/assets/0014b531-8a95-4f1c-8684-f7c7bd71ee4a)

https://github.com/user-attachments/assets/180084bf-1ab3-4e42-8291-3dfa693e1b79

I could see it in the latest build as well

It only happens at particular width

**Comment by roblourens** [comment]
Hm, I still can't repro. Are you zoomed in, or any other unusual settings? Is this windows or linux?

**Comment by abhijit-chikane** [comment]
No zooming
OS Windows 11

**Comment by abhijit-chikane** [comment]
Closing this as It is it is no more reproducible
Thanks
